settings.gradle

```diff
@@ -1,4 +1,4 @@
 rootProject.name = 'terra-aws-resource-discovery'
 
 gradle.ext.projectGroup='bio.terra'
-gradle.ext.releaseVersion='vv0.0.1-SNAPSHOT'
```

Comment:
@dexamundsen am I doing the right thing by bumping this manually, or should I leave this out of the PR and use the GHA workflow trigger after this merges?

Also, I think this would normally be a minor bump per semver (since we're adding a new method), but keeping it minor since we're still in major version 0.

Reply:
Bump handled in tag workflow automatically. There is a bug in DSP/action, fix is in - https://github.com/DataBiosphere/github-actions/pull/33

Reply:
removing this change manually from your branch. 
i will enable auto publish in my PR